canonical-web-and-design/maas-ui
# Point all machine listing routes to the react machine listing.

## Done
* Point all machine listing routes to the react machine listing.

## QA
* Run app, login. You should be redirected to `/r/machines` by default.
* Ensure links back to the machine listing are routed to the react app.
